modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/configuration/CTConfiguration.java

```diff
@@ -43,4 +43,11 @@ public interface CTConfiguration {
 	)
 	public String[] unsupportedApplication();
 
+	@Meta.AD(
+		deflt = "production",
+		description = "replaces-usages-of-the-term-'production'-with-a-custom-name",
+		name = "custom-production-name", required = false
+	)
+	public String customProductionName();
```

Comment:
We need to change the return type to `LocalizedValuesMap` to allow customers to add their own translations for the replacement term. 

Example: [AccountEntryEmailConfiguration](https://github.com/liferay/liferay-portal/blob/master/modules/apps/account/account-api/src/main/java/com/liferay/account/configuration/AccountEntryEmailConfiguration.java#L32)
<img width="1106" alt="Screen Shot 2023-12-04 at 7 00 59 PM" src="https://github.com/liferay-publications/liferay-portal/assets/5168351/96d3e9ab-ecff-42a5-bedc-6200281ec95d">